### PR TITLE
Recovery

### PR DESCRIPTION
**Pull request by patricknitschke**
Added utilites/recovery folder:
- recovery.h/c has two globals where get-functions used in airbrakes, drogue and chute states. Parachute global has bools to check if drogue, main chute deployed. ApogeeArray in other file.
- apogee_detect.h/c used to detect apogee in airbrakes state. Detect function in .cpp, uses total acceleration and altitude with a timer to detect apogee. (releases drogue after 3s when acc < 0.3)
Changes to airbrakes, apogee, drogue and chute states:
- airbrakes: changed state transition code
- apogee: deploys drogue immediately
- drogue: checks if altitude < 457m and deploys main
- chute: possibility to stops rotation of continuous servo, if we need to.

### CODE REVIEW

**Review by Kevinkald — CHANGES_REQUESTED** [review]
1. Please pull master once and then make a new pull request.
2. In master #define START_STATE AIRBRAKES should be set to #define START_STATE IDLE
3. Is it necessary to init servos inside the flight_computer sketch? Might be nice to add these in to the newly created servo_interface library.

**Comment by patricknitschke** [comment]
Alright, I've pulled master and changed START_STATE to idle. Yeah the servos uses the servo_interface library, I just added the init servo code below airbrake's in the flight_computer sketch.

**Comment by patricknitschke** [comment]
Might be smarter to init the servos directly on start up instead of inside one of the states, but we can init the servos in the first state if we want to.

File: flight_computer.ino [hunk 1]
```diff
@@ -9,6 +9,7 @@
 */
 
 #include <Wire.h>
+#include <SD.h>
```
Comment (OleB) [inline flight_computer.ino]: Do we need this?

Comment (Kevinkald) [reply flight_computer.ino]: This is not included in the master branch, looks like you have not pulled master and resolved conflicts.

Comment (patricknitschke) [reply flight_computer.ino]: Alright, I've pulled again but it didn't change, so I've manually removed it and pushed the changes.

**Review by Kevinkald — APPROVED** [review]

**Pull request merged**